language/tools/move-mv-llvm-compiler/src/main.rs

```diff
@@ -17,107 +18,131 @@ use move_bytecode_source_map::{mapping::SourceMapping, utils::source_map_from_fi
 use move_command_line_common::files::{
     MOVE_COMPILED_EXTENSION, MOVE_EXTENSION, SOURCE_MAP_EXTENSION,
 };
+use move_compiler::shared::PackagePaths;
 use move_ir_types::location::Spanned;
+use move_model::{model::GlobalEnv, run_bytecode_model_builder, run_model_builder};
 use move_mv_llvm_compiler::{cli::Args, disassembler::Disassembler};
-use std::{fs, path::Path};
+use std::{f32::consts::E, fs, path::Path};
```

Comment:
What is this constant for? I didn't notice it being used.

Reply:
It was "automatically" added by VS Code. Admittedly, I had more code at some point in development. 
At the end I run our cargo utilities before the last commit and this line was not reported. But I'll double check if I can remove this extra.